### ISSUE TITLE
[NPC][WinterSprings] Elder Shardtooth - Sometimes without loot

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4625]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Few things to mention. 
About half the time these bears drop no loot at all. 
The second issue being the drop rate on bear flanks seems lower. This could be due to that some don't drop any loot.
**Expected behaviour**: Tell us what should happen instead
Kill bear get something.
**Steps to reproduce the problem**:

1. Kill Elder Shardtooths
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/
NPC Loctions seems fine
http://wotlk.openwow.com/npc=7445

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
the refernce_loot table is broken there

**Comment by TheRealestMcCoy** [comment]
So this cant be fixed or?

**Comment by Rushor** [comment]
it can, don't worry :)

**Comment by TheRealestMcCoy** [comment]
Didnt know if it would be worth reporting any more of these that may have drop issues or not then .

**Comment by TheRealestMcCoy** [comment]
@Rushor also BTW do they still need help testing fixes on the PTR?

**Comment by Rushor** [comment]
we can always need testers :)

as you can see we have https://github.com/dalaranwow/dalaran-wow/issues?q=is%3Aissue+label%3A%22Fixed+-+On+PTR%22+is%3Aclosed **265** tickets that need to be tested on PTR and **960** https://github.com/dalaranwow/dalaran-wow/issues?q=is%3Aissue+is%3Aclosed+label%3A%22Fixed+-+Waiting+for+PTR+Update%22 issues that need to be applied on PTR first and then need to be tested :D

**Comment by TheRealestMcCoy** [comment]
Well I sent a email to the email address they had for the tester help on the website. if you want to pass it along.

**Comment by Rushor** [comment]
DB/Creature: Elder Shardtooth - now always contains at least on item

* the creature now always drops one of those items: 11408, 11409, 11410 additionally to the other loot
* closes: https://github.com/dalaranwow/dalaran-wow/issues/4625
* commit: d05f3d7722315a8a7a8922a2d42b2d07111f499b

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
